src/main/resources/db/changeLog.xml

```diff
@@ -125,7 +125,7 @@
                 <constraints primaryKey="true" nullable="false"/>
             </column>
             <column name="vacationrequest_id" type="binary(16)">
-                <constraints foreignKeyName="FK_vacationrequest_id" referencedTableName="vacationRequest" referencedColumnNames="id"/>
+                <constraints foreignKeyName="FK_vacationrequest_id" referencedTableName="VacationRequest" referencedColumnNames="id"/>
```

Comment:
Revert this change. Migrations that are already applied in any real database should not be touched. Instead, configure your development database to run in case insensitive mode.

Reply:
Reverting this change. Will have to find a way for the case insensitive mode.